osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/entities/SecurityKeyType.java

```diff
@@ -25,7 +25,7 @@
   /** M-Bus User key */
   G_METER_ENCRYPTION(SecretType.G_METER_ENCRYPTION_KEY),
   /** M-Bus Firmware update authentication key */
-  G_METER_FIRMWARE_UPDATE_AUTHENTICATION(SecretType.G_METER_FIRMWARE_UPDATE_AUTHENTICATION_KEY),
+  G_METER_FIRMWARE_UPDATE(SecretType.G_METER_FIRMWARE_UPDATE_KEY),
```

Comment:
The renaming of SecretKeyType G_METER_FIRMWARE_UPDATE_AUTHENTICATION_KEY to G_METER_FIRMWARE_UPDATE_KEY will be rolled back as it will not follow the exact naming in de SMR5 documentation

Reply:
Rolled back the renaming and resize secret_type column in encrypted_secret table in secret_management database to 42